scripts/build/ci-build/Dockerfile

```diff
@@ -155,7 +155,11 @@ RUN cd /tmp && \
     tar xf x86_64-linux-musl-cross.tgz && \
     rm x86_64-linux-musl-cross.tgz
 
-RUN go get -u github.com/mgechev/revive@v1.0.2
-RUN mv ${GOPATH}/bin/revive /usr/local/bin/
+RUN go get -u github.com/mgechev/revive@v1.0.2 && \
```

Comment:
how often do we rebuild this image? I dont know how frequent changes we have for mixtool tbh

Reply:
@bergquist We don't have a particular schedule for it. If it should need frequent updates, we should automate rebuilding it via Drone. BTW, I ran into a problem installing mixtool with the -u flag, since some unkwon package has a resolution issue. That shows how important it is to install dependencies in the build image and not during each build, since it's extremely fragile and can stop our builds.